server/src/server.rs

```diff
@@ -231,6 +235,26 @@ pub enum BatchRequestConfig {
 	Unlimited,
 }
 
+/// Configuration for batch request handling.
+#[derive(Debug, Copy, Clone)]
+pub enum PingConfig {
```

Comment:
please add `Disabled` here as well

Reply:
nah let's do in another PR

Reply:
Should we make ping_interval optional then?

Reply:
What do you mean by "optional"?

`ping_interval` has a default value and anyone that wants another behavior can just call `ping_interval(PingConfig)` but I guess you mean disable it by default?